jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -110,6 +111,7 @@ def _count_results(self, url):
         return len(content.get('results', 0))
 
 
+@freeze_time('1970-01-01 00:00:00')
```

Comment:
One question, do you feel that the tests running slows down when using `freeze_time`? This happened with me, but I've never profiled to check if this was happening.